addons/mail/tests/test_mail_activity.py

```diff
@@ -147,38 +147,65 @@ def _instantiate_activity_schedule_wizard(self, records, additional_context_valu
         }))
 
 
-@tagged("-at_install", "post_install")
+@tagged("-at_install", "post_install", "mail_activity")
 class TestMailActivityChatter(HttpCase):
 
-    def test_mail_activity_schedule_from_chatter(self):
-        testuser = self.env['res.users'].create({
-            'email': 'testuser@testuser.com',
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.test_partner = cls.env['res.partner'].create({
+            'email': 'test.partner@example.com',
             'name': 'Test User',
-            'login': 'testuser',
-            'password': 'testuser',
         })
-        self.start_tour(
-            f"/web#id={testuser.partner_id.id}&model=res.partner",
-            "mail_activity_schedule_from_chatter",
-            login="admin",
-        )
 
     def test_mail_activity_date_format(self):
         with freeze_time("2024-1-1 09:00:00 AM"):
             LANG_CODE = "en_US"
             self.env = self.env(context={"lang": LANG_CODE})
-            testuser = self.env['res.users'].create({
-                "email": "testuser@testuser.com",
-                "name": "Test User",
-                "login": "testuser",
-                "password": "testuser",
-            })
             lang = self.env["res.lang"].search([('code', '=', LANG_CODE)])
             lang.date_format = "%d/%b/%y"
             lang.time_format = "%I:%M:%S %p"
 
             self.start_tour(
-                f"/web#id={testuser.partner_id.id}&model=res.partner",
+                f"/web#id={self.test_partner.id}&model=res.partner",
                 "mail_activity_date_format",
                 login="admin",
             )
+
+    def test_mail_activity_schedule_from_chatter(self):
+        self.start_tour(
+            f"/web#id={self.test_partner.id}&model=res.partner",
+            "mail_activity_schedule_from_chatter",
+            login="admin",
+        )
+
+
+@tagged("-at_install", "post_install", "mail_activity")
+class TestMailActivityIntegrity(ActivityScheduleCase):
+
+    def test_mail_activity_type_master_data(self):
+        """ Test master data integrity
```

Comment:
Note that it'd be interesting to test one use case where you can write and one use case where you can delete, to confirm it still works.